### PR TITLE
TMDM-11900 Update schema of UpdateReport, add 'UUID' field for standalone mode.

### PR DESCRIPTION
**Pull request by sbliu**
https://jira.talendforge.org/browse/TMDM-11900

### CODE REVIEW

File: main/plugins/org.talend.repository.mdm/resources/UpdateReport.xsd [hunk 1]
```diff
@@ -11,6 +11,7 @@
 			<xsd:element maxOccurs="1" minOccurs="0" name="UserName" nillable="true" type="xsd:string"/>
 			<xsd:element maxOccurs="1" minOccurs="1" name="Source" nillable="false" type="xsd:string"/>
 			<xsd:element maxOccurs="1" minOccurs="1" name="TimeInMillis" nillable="false" type="xsd:string"/>
+			<xsd:element maxOccurs="1" minOccurs="1" name="UUID" nillable="false" type="xsd:string"/>
```
Comment (VictorLi-talendbj) [inline main/plugins/org.talend.repository.mdm/resources/UpdateReport.xsd]: Do we also need to have `UUID` in standalone?

Comment (sbliu) [reply main/plugins/org.talend.repository.mdm/resources/UpdateReport.xsd]: As task requirements, standalone mode only need add UUID element, cluster mode will change more, here only consider standalone mode change, so add UUID element .

Comment (sbliu) [reply main/plugins/org.talend.repository.mdm/resources/UpdateReport.xsd]: This PR could be merged at same time when related change in mdm part merged.

Comment (VictorLi-talendbj) [reply main/plugins/org.talend.repository.mdm/resources/UpdateReport.xsd]: Is it ok to have nillable="true" for UUID?

Comment (lpw82gf) [reply main/plugins/org.talend.repository.mdm/resources/UpdateReport.xsd]: No, it should be `false`,  we will set that value for sure. And if not, we can't switch from standalone to cluster.

**Pull request merged**